simple_history/management/commands/populate_history.py

```diff
@@ -54,14 +57,16 @@ def handle(self, *args, **options):
             for model in models.registered_models.values():
                 try:    # avoid issues with mutli-table inheritance
                     history_model = utils.get_history_model_for_model(model)
-                except utils.NotHistorical:
+                except NotHistoricalModelError:
                     continue
                 to_process.add((model, history_model))
             if not to_process:
-                self.stdout.write(self.NO_REGISTERED_MODELS)
+                if self.verbosity >= 1:
+                    self.stdout.write(self.NO_REGISTERED_MODELS)
```

Comment:
Is `stdout` the usual convention for Django library logging?

Reply:
Yep